### PR TITLE
Add tips for re-registering blocks from Prefect Collections

### PR DESCRIPTION
**Pull request by discdiver**
### Checklist

- [x] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [ ] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`

### CODE REVIEW

**Comment by discdiver** [comment]
closes #8323

File: docs/collections/usage.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ As an example, to register the blocks available in `prefect-aws`:
 prefect block register -m prefect_aws
 ```
 
-**Note**, to use the load method on Blocks, you must already have a block document [saved](/concepts/blocks/#saving-blocks) through code or saved through the UI.
+**Note**, if you install an updated Prefect collection that adds fields to a block type, you will need to re-register that Block type.
```
Comment (tpdorsey) [inline docs/collections/usage.md:34]:
```suggestion
!!! tip "Updating blocks from Prefect Collections"
    If you install an updated Prefect collection that adds fields to a block type, you will need to re-register that block type.
```

Comment (tpdorsey) [reply docs/collections/usage.md:34]: Let's make these admonitions to raise visibility.

File: docs/collections/usage.md [hunk 2]
```diff
@@ -31,7 +31,9 @@ As an example, to register the blocks available in `prefect-aws`:
...
+
+**Note**, to use the load method on a Block, you must already have a block document [saved](/concepts/blocks/#saving-blocks) through code or saved through the UI.
```
Comment (tpdorsey) [inline docs/collections/usage.md:36]:
```suggestion
!!! tip "Loading a block in code"
    To use the `load` method on a Block, you must already have a block document [saved](/concepts/blocks/#saving-blocks) either through code or through the Prefect UI.
```

**Review by tpdorsey — APPROVED** [review]

**Pull request merged**